ts/nni_manager/common/log.ts

```diff
@@ -75,15 +75,17 @@ export class Logger {
             return;
         }
 
-        // `time.toLocaleString('sv')` trick does not work for Windows
-        const isoTime = new Date(new Date().toLocaleString() + ' UTC').toISOString();
-        const time = isoTime.slice(0, 10) + ' ' + isoTime.slice(11, 19);
+        const zeroPad = (num: number): string => num.toString().padStart(2, '0');
+        const now = new Date();
+        const date = now.getFullYear() + '-' + zeroPad(now.getMonth() + 1) + '-' + zeroPad(now.getDate());
+        const time = zeroPad(now.getHours()) + ':' + zeroPad(now.getMinutes()) + ':' + zeroPad(now.getSeconds());
+        const datetime = date + ' ' + time;
```

Comment:
Looks really tedious. But I guess there's no better way.

Reply:
Yes. The only better way may be using a third-party `strftime` library.